buku.py

```diff
@@ -67,6 +67,16 @@
 logdbg = logger.debug
 logerr = logger.error
 
+def read_input(msg):
```

Comment:
An example is this line

`resp = read_in('Delete specified tag(s) from ALL bookmarks? (y/n): ')`

81 characters, not really sure how to break it.